server/models/default_local_provider.go

```diff
@@ -1546,7 +1508,9 @@ func getSeededComponents(comp string, log logger.Handler) ([]string, []string, e
 			return nil, nil, er
 		}
 	}
-	if !viper.GetBool("SKIP_DOWNLOAD_CONTENT") {
+	if comp == "Application" && viper.GetBool("SKIP_DOWNLOAD_APPLICATIONS") {
+		log.Info("Skipping seeding sample applications")
+	} else if !viper.GetBool("SKIP_DOWNLOAD_CONTENT") {
```

Comment:
@vaishnaviadhau we don't need to log that we are skipping seeding applications , you can remove it and also `SKIP_DOWNLOAD_APPLICATIONS` why we need this?

Reply:
seeding of sample applications is being skipped, and the subsequent code to download and seed applications is not executed.